src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -0,0 +1,61 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.VictorSPX;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class Elevator extends SubsystemBase {
+
+  //creates instance of victorspx motor controller for elevator motor
+  VictorSPX motor = new VictorSPX(Constants.elevatorPort);
+
```

Comment:
Could I have a brief comment explaining what this variable does?

Reply:
redid the comment

Reply:
I meant the toggle boolean. the comment on the motor was fine.